app/grants/tasks.py

```diff
@@ -207,6 +207,9 @@ def process_grant_contribution(self, grant_id, grant_slug, profile_id, package,
             # https://gitcoincore.slack.com/archives/C01FQV4FX4J/p1607980714026400
             if not settings.DEBUG:
                 subscription.network = 'mainnet'
+            else:
+                # Truncate the field length as the max length in DB is 8 at this time
+                subscription.network = "undef"
```

Comment:
not sure what's going on here... is this used somewhere else?

Reply:
@chibie network was defined to have max_length=8, "undefined" would never have been saved there anyway. I just changed it to "undef" to avoid errors when testing.
Alternatively we could change the field definition? Would this be better?

```
    network = models.CharField(
        max_length=8,
        default='mainnet',
        db_index=True,
        help_text=_('The network in which the Subscription resides.'),
    )
    ```